README.md

```diff
@@ -118,6 +118,7 @@ Unsure what to contribute? Check out the `good first issue` [tagged Github issue
 - [ArrowVortex](https://arrowvortex.ddrnl.com/) - Create or edit stepfiles for various rhythm games, such as [DDR (Dance Dance Revolution)](https://www.ddrgame.com/), [ITG (In The Groove)](https://en.wikipedia.org/wiki/In_the_Groove_(video_game)), [PIU (Pump It Up)](https://www.piugame.com/piu.xx/), [StepMania](https://www.stepmania.com/) and [osu!](https://osu.ppy.sh/home)
 - [fnf-to-sm](https://github.com/Ashen-Haze/fnf-to-sm) - A fork of a [fork](https://github.com/KadeDev/fnf-to-sm) from the original [fnf-to-sm repository](https://github.com/shockdude/fnf-to-sm). Converts Funkin' .json charts to StepMania simfiles and vice versa.
```

Comment:
Doesn't *this* also have Dance Double?

Reply:
Both fnf to sm and this one use dance double sm charts to convert to fnf, however fnf to sm doesn't convert bpm changes

Reply:
It (fnf-to-sm) can for convert an fnf chart with bpm changes to sm (usually comes out very broken though) but not the other way around

Reply:
Should the one on line 119 be removed then?

Reply:
Gonna say rq that no cause even if i do find it to be an alternative, it is a bit less convenient since it's not as simple as a drag and drop like in fnf-to-sm